packages/hooks/src/hooks/useWallet.test.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useWallet } from './useWallet';
+import { Provider } from '../Provider';
+import { renderHook } from '@testing-library/react-hooks';
+
+let windowSpy;
+
+beforeEach(() => {
+  windowSpy = jest.spyOn(window, 'window', 'get');
+});
+
+afterEach(() => {
+  windowSpy.mockRestore();
+});
+
+describe('useWallet tests', () => {
+  test('should use the wallet', () => {
+    const wrapper = ({ children }) => (
+      <Provider network={1} infuraId={'abc123'}>
```

Comment:
this is typically how react passes strings as props

```suggestion
      <Provider network={1} infuraId="abc123">
```

Reply:
agreed. been using something that has gotten me in the habit of objectifying my strings and i shouldn't.